src/hide.rs

```diff
@@ -63,7 +63,7 @@ mod nightly {
 }
 
 // When a C compiler is available, a dummy C function can be used.
-#[cfg(not(feature = "no_cc"))]
+#[cfg(all(not(feature = "no_cc"), feature = "std"))]
 mod cc {
     use std::os::raw::c_void;
```

Comment:
On second thought, the `std` feature I added exists exclusively to support this. It'd be nice to not have a `std` feature at all.

An interim solution would be to change the `no_cc` feature to be a default `cc` feature (features are supposed to be additive anyway), and have an optional dependency on `libc` pulled in when the `cc` feature is enabled.